### ISSUE TITLE
Why the p-cnn features I extract are different with author's?

### ISSUE THREAD

**Issue by siyilingting** [gcheron/P-CNN#1]
First of all, thanks the author for making the code publicly available.
  I have downloaded the JHMDB dataset which contains 928 videos, jhmdb joint positions and splits. And I splited them following the illustration. I also got raw features( Xn_train.mat, Xn_test.mat ) and linear kernel( Ktrain, Ktest), but I noticed that the Ktest I have extracted is different with the Ktest provided by author in "ICCV15_results\ktest_split1.mat" , of course it's also defferent with ktest_split2.mat and ktest_split3.mat. 
I upload 2 pictures, which are parts of ktest. The first picture is author's kernel and the second is the kernel I extract.  I am looking forward to your help. Thanks.
![ktest_author](https://cloud.githubusercontent.com/assets/23379451/21094208/c1cb4a9e-c090-11e6-8449-1c2d7ecd9e0e.png)
![ktest_me](https://cloud.githubusercontent.com/assets/23379451/21094209/c1d1dbde-c090-11e6-859e-b114e695a941.png)

**Comment by gcheron** [comment]
Hi,
The kernel you obtain seems relatively close to the one I provide. Small differences can happen for some reasons like: OF computation, numerical precision, etc... Have you try to evaluate the performance with your test kernels? Is it close to the one obtained with the provided kernels?

**Comment by siyilingting** [comment]
@gcheron 
Hi,
Thanks for your response. At first I evaluate the performance with the SVMs you provided and test kernels I got, but the results is 73.5%. Then I used the train kernels I got to train the SVM by libsvm, and the parameter of svmtrain() is '-t 4'. And the accuracy with the new SVMs is 74.1%. I put the picture of results here. I'm not sure whether the results are normal. I'm looking forward to your advice. Thank you very much ! ! !
![results](https://cloud.githubusercontent.com/assets/23379451/21312983/1b0f346e-c62a-11e6-8082-d49ae839fca0.png)
